GTHalfSheet/GTSheet/Utilities/Protocols.swift

```diff
@@ -37,7 +37,7 @@ internal extension Array where Element == DismissMethod {
 }
 
 public protocol HalfSheetPresentableProtocol: class {
-    weak var managedScrollView: UIScrollView? { get }
+    var managedScrollView: UIScrollView? { get }
```

Comment:
context: Swift 5 deprecates weak members on protocols because apparently they were not enforced by the compiler anyway. sad.